src/microbe_stage/PatchMap.cs

```diff
@@ -12,6 +12,13 @@ public class PatchMap : ISaveLoadable
 {
     private Patch? currentPatch;
 
+    public enum FogOfWarMode
+    {
+        Disabled,
+        Light,
```

Comment:
Maybe this could be named "normal"? I'm very in favour of this being the normal mode so it would make sense to the players seeing it for the first time (and also helps if the term used in the game text is the same as in the code).